docs/usage_guide.rst

```diff
@@ -367,6 +367,15 @@ To stop tracking a particular account scroll down to the accounts tables and cli
 
 If an ethereum account also contains tracked tokens you can click on the arrow under "Actions" in order to expand its view and show the balance breakdown for the account over all assets it holds.
 
+For Bitcoin you can add addresses manually or let rotki discover them using a public key. From this key rotki knows how to generate your addresses and can query the Bitcoin's blockchain generating them until it finds an address without transactions.
+The different formats of allowed keys refer to how the addresses are generated, "P2PKH" uses keys starting with "xpub", "P2SH_P2WPKH" uses keys starting with "ypub" and "WPKH" uses keys starting with "zpub".
```

Comment:
That is not correct. This is just a convention. For example ledger live WILL NOT follow this convention. While the electrum wallet will. It's a confusing mess.

This is why you as a user **have** to tell us what type of xpub it is. If it starts with zpub, or ypub you don't. We can see that it follow the convention and that's great. But for say xpubs generated by ledger live (all starting with `xpub` irrespective of type) we need this extra info.